mex-wholebodymodel/library/src/mexmain.cpp

```diff
@@ -39,68 +35,56 @@ using namespace mexWBIComponent;
  * To properly destroy, call ComponentManager::deleteInstance()
  * and then set this pointer to 0.
  */
-static ComponentManager *componentManagerLocalPointerCopy = 0;
```

Comment:
Please, do not change `0` to `NULL`. In C++98, the way of indicating a null ptr is indeed `0` (in C++11 it has been superseded by `nullptr`. See http://www.stroustrup.com/bs_faq2.html#null .